packages/inter-protocol/test/auction/test-auctionBook.js

```diff
@@ -85,7 +94,13 @@ test('simple addOffer', async t => {
   const initialPrice = makeRatioFromAmounts(moola(20n), simoleans(100n));
   const pa = buildManualPriceAuthority(initialPrice);
 
-  const book = await makeAuctionBook(moolaKit.brand, simoleanKit.brand, pa);
+  const publishKit = makePublishKit();
```

Comment:
that the whole kit is kept makes me look for how the subscriber is used, but it isn't. please take only what you need. POLA-ish.

```suggestion
  const { publisher }  = makePublishKit();
```

Reply:
done